README.md

```diff
@@ -92,9 +92,16 @@ If the `aws_tag_select` feature was used, an additional information metric will
 # TYPE aws_resource_info gauge
 aws_resource_info{job="aws_elb",instance="",arn="arn:aws:elasticloadbalancing:eu-west-1:121212121212:loadbalancer/mylb",load_balancer_name="mylb",tag_Monitoring="enabled",tag_MyOtherKey="MyOtherValue",} 1.0
 ```
-
 All metrics are exported as gauges.
 
+The `aws_resource_info` can be used in conjunction with other metrics to expose tags using PromQL syntax like:
+```
+  aws_elb_request_count_sum
+*
+  on(load_balancer_name) aws_resource_info
+{load_balancer_name="myotherlb"} 7.0
```

Comment:
this should not be part of the same pre, it reads as though this is part of the expression